package.json

```diff
@@ -38,6 +38,6 @@
     "nyc": "^15.1.0"
   },
   "engines": {
-    "node": ">=8.6.0"
+    "node": ">=10.13.0"
```

Comment:
Node 10 reaches EOL at Apr 30. Consider bump to `>=12.17.0`, which has native ESM support.

Reply:
Thank you :)

Yes, I know. But, the current [hexojs/hexo](https://github.com/hexojs/hexo) is not yet dropped v10. 
IMHO we should not bump to `12.x` until [hexojs/hexo](https://github.com/hexojs/hexo) drop `v10` 😓 

> https://github.com/hexojs/hexo/blob/eef2eac874b5118cae73fcdb4acba3a6c744f2b5/package.json#L80

Reply:
`hexo-util` now requires Node.js 12.4.0 https://github.com/hexojs/hexo-util/blob/576fdd6a0c8b16be4cb63f1c1f38ee1f9508dc24/package.json#L56

Reply:
@stevenjoezhang 
Done :)